palindrome_checker.cpp

```diff
@@ -1,6 +1,43 @@
+/*
+______________ Assumptions _____________
```

Comment:
Thanks for including your TEBOW IT notes and your assumptions (i.e. the talk step)!